### PR TITLE
[RO-3870] Change the PR job to non-artifacted

### PR DESCRIPTION
**Pull request by cloudnull**
The default PR test needs to be non-artifacted. This change updates the
PR image properties so that non-artifacted jobs can run as the default.

Signed-off-by: Kevin Carter <kevin.carter@rackspace.com>

Issue: [RO-3870](https://rpc-openstack.atlassian.net/browse/RO-3870)

### CODE REVIEW

File: rpc_jobs/rpc_openstack.yml [hunk 1]
```diff
@@ -174,7 +174,6 @@
     branch: "master"
     jira_project_key: "RO"
     image:
-      - xenial_mnaio
```
Comment (prometheanfire) [inline rpc_jobs/rpc_openstack.yml]: why was this removed?

Comment (odyssey4me) [reply rpc_jobs/rpc_openstack.yml]: Strict artifact builds using MNAIO may or may not work depending on when upstream packages are released versus the last successful build because the MNAIO builds deploy the base OS using the latest upstream packages, not the artifact repo. As such, this test is a bit pointless until we get the base OS deployment working with the artifacts.

Comment (prometheanfire) [reply rpc_jobs/rpc_openstack.yml]: ok
